src/test/java/com/google/sps/data/DatastoreScheduledInterviewDaoTest.java

```diff
@@ -123,19 +130,68 @@ public void getsAllScheduledInterviews() {
             result.get(0).id(),
             scheduledInterview1.when(),
             scheduledInterview1.interviewerId(),
-            scheduledInterview1.intervieweeId());
+            scheduledInterview1.intervieweeId(),
+            scheduledInterview1.shadowId());
     ScheduledInterview copyScheduledInterview2 =
         ScheduledInterview.create(
             result.get(1).id(),
             scheduledInterview2.when(),
             scheduledInterview2.interviewerId(),
-            scheduledInterview2.intervieweeId());
+            scheduledInterview2.intervieweeId(),
+            scheduledInterview2.shadowId());
     List<ScheduledInterview> expected = new ArrayList<ScheduledInterview>();
     expected.add(copyScheduledInterview1);
     expected.add(copyScheduledInterview2);
     Assert.assertEquals(expected, result);
   }
 
+  // Tests whether all scheduledInterviews for a particular user when they are interviewee are
+  // retrieved.
+  @Test
+  public void getForPersonAsInterviewee() {
```

Comment:
I think this and the next test are kind of unneccesary and could be replaced with one test where the user fills all 3 roles in the interviews returned.

Reply:
Will replace